chain/client/src/client_actor.rs

```diff
@@ -179,8 +185,27 @@ impl ClientActor {
             chunk_request_retry_next_attempt: now,
             sync_started: false,
             state_parts_task_scheduler: Box::new(move |msg: ApplyStatePartsRequest| {
-                if let Err(_) = state_parts_actor_addr.try_send(msg) {
-                    panic!("Can't send message to StatePartsActor");
+                if let Err(err) = sync_jobs_actor_addr.try_send(msg) {
```

Comment:
I think you don't need the `move` here, capturing by reference seems fine to me. Then you don't need to clone the address.

Reply:
But I think this closure outlive any address object because it will be destroyed at the end of new method? I may be wrong here

Reply:
Ah that is true